[2022-11-30T09:26:34.413+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:26:31.588626+00:00 [queued]>
[2022-11-30T09:26:34.422+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:26:31.588626+00:00 [queued]>
[2022-11-30T09:26:34.423+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T09:26:34.423+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-30T09:26:34.423+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T09:26:34.430+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonSensor): waiting_for_data> on 2022-11-30 09:26:31.588626+00:00
[2022-11-30T09:26:34.433+0000] {standard_task_runner.py:55} INFO - Started process 298 to run task
[2022-11-30T09:26:34.436+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pratice_pg', 'waiting_for_data', 'manual__2022-11-30T09:26:31.588626+00:00', '--job-id', '709', '--raw', '--subdir', 'DAGS_FOLDER/practice_pg.py', '--cfg-path', '/tmp/tmpyikk_2yf']
[2022-11-30T09:26:34.436+0000] {standard_task_runner.py:83} INFO - Job 709: Subtask waiting_for_data
[2022-11-30T09:26:34.474+0000] {task_command.py:376} INFO - Running <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:26:31.588626+00:00 [running]> on host 481f60569de1
[2022-11-30T09:26:34.515+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pratice_pg
AIRFLOW_CTX_TASK_ID=waiting_for_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-30T09:26:31.588626+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-30T09:26:31.588626+00:00
[2022-11-30T09:26:34.520+0000] {python.py:68} INFO - Poking callable: <function pg_check_data at 0x7fae1734cc20>
[2022-11-30T09:26:34.525+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T09:26:34.627+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T09:26:34.631+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-30T09:26:34.636+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=pratice_pg, task_id=waiting_for_data, execution_date=20221130T092631, start_date=20221130T092634, end_date=20221130T092634
[2022-11-30T09:26:34.653+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T09:26:34.671+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
